source/grid/grid_tools_dof_handlers.cc

```diff
@@ -592,15 +592,12 @@ namespace GridTools
                                      const double                   tolerance,
                                      const std::vector<bool> &marked_vertices)
   {
-    // first use the result of the single point function as a guess. In order
-    // not to make the other find_all_active_cells_around_point more expensive
-    // and avoid some additional logic there, we first start with one cell as
-    // given by that other function (that possibly goes through a larger set
-    // of cells) and later add a list of more cells as appropriate.
     std::vector<
       std::pair<typename MeshType<dim, spacedim>::active_cell_iterator,
                 Point<dim>>>
       cells_and_points;
+
+    // find a first cell and point
     try
       {
         cells_and_points.push_back(find_active_cell_around_point_tolerance(
```

Comment:
How about rewriting this part slightly, i.e., remove the `cells_and_points` vector above and inside the try clause do
```c++
const auto cell_and_point = find_active_cell_around_point_tolerance(...);
return find_all_active_cells_around_point(
          mapping, mesh, p, tolerance, cell_and_point);
```
and as a default case (search did not succeed) return `{}`? I think that would make the code easier to read.

Reply:
I addressed these points. I am not sure about the ifdef in the declaration because for the new function `MeshType` is now also a parameter of the function. Do I also have to add something to a .inst.in file?